### DISCUSSION TITLE
Invite external collaborator to projects (beta)

### DISCUSSION THREAD

**Discussion by varunsrin** [community/community#5874] [Projects and Issues] [answered]
Currently, it doesn't seem possible to invite an external collaborator to a project made with project (beta). Are there plans to add this in the near future?

For project boards (old version), the way to do this is to go to Project -> Settings -> Collaborators and invite people there

**Comment by zaporylie** [comment]
I have the same problem - I fail to add an external collaborator to Project (beta). Would it be possible to get a clarification from someone on the Github team if that's a known limitation or a regression (bug)? As someone who's working with several consultants (external collaborators), I really hope we can get this feature in place asap.
Thanks for all the work you've already put into this - it's starting to shape as something really cool which I'll depend on once the remaining issues get resolved!

**Comment by 2shrestha22** [comment]
I can invite user but no invitation email was arrived so unable to add the user.

**Comment by francoislauger** [comment]
Same probleme here, we were going to use the new Project [Beta], but that's will not be possible if outside colaborators can't be add to private projects.

Outside collaborator can view/edit the issues in the repo, but can't acces the Project [Beta].

**Comment by davidpaulsson** [comment]
Same issue! Collaborator added to the repo, but can't access our project :(

**Comment by azenMatt** [comment]
Hi everyone - we are working on this at the moment and should have some updates in February for you 😄

**Reply by azenMatt** [reply]
We are currently in a limited roll our of outside collaborators - if you would like to get early access please let me know your org name and I can add you in 😄

**Reply by phil-opp** [reply]
The 02/02 update post at https://github.blog/changelog/2022-02-02-the-new-github-issues-02-02-update/ mentions the following:

> - [Outside collaborators](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/adding-outside-collaborators-to-repositories-in-your-organization) can now be added to projects. Outside collaborators who have already been added to at least one repository in your organization will now appear when searching for users in the `Manage access` settings page.

Does that mean that this feature is now available to all organizations?

**Reply by azenMatt** [reply]
Hi @phil-opp - yes! that is correct! Let me know if it works well for you!

**Reply by plourenco** [reply]
Thanks @azenMatt, this works just fine. Just to add some feedback: we had to invite a couple of externals as collaborators of our organization and then give them write access in the board. While this surely works, it would be much easier if we could directly add them through the board. I'm not exactly sure why Github restricts the invitation of board collaborators to organization collaborators but it would be great as a suggestion if this "external" logic was built-in within each board.

**Comment by azenMatt** [comment, marked as answer]
This was included in our latest release -  https://github.blog/changelog/2022-02-02-the-new-github-issues-02-02-update 🎉

**Reply by PaulLeitner** [reply]
Hi @azenMatt - inviting (Outside) Collaborators doesn't seem to work with Personal Pro accounts though it seems - is that true? I am reluctant in openeing an organizational account because I fear that things we have setup already might not be available / work any more then.

**Reply by azenMatt** [reply]
@PaulLeitner - you need the collaborator to have access to at least one repo under the account (so that they can see the issues /prs in the project. Does that help?

**Comment by appliedrd** [comment]
Project invite  : no email sent
I am a pro user trying to invite free users to my project.  When I invite them, no email is sent/received. 

1. invite and accept collaborator on a repo
2.  created a new private project. 
3. settings ->manage access -> 'invite collaborators' -> Invite. 
4. user is listed on project, but no email sent

**Comment by yizivan** [comment]
I am facing the same issue while trying to invite other users to my project. They never could access the project even though they are listed on my project access members

**Comment by PaulLeitner** [comment]
Thanks Matt,

the collaborators have access to a Repo under the account (linked to an old Project) but this Repo is not linked to the Project Beta – as I can’t manage to link it as mentioned.
Is that sufficient?
How can I link the Repo to a Project Beta – the instructions given on Github are not up to date for Project Beta?

https://docs.github.com/en/issues/organizing-your-work-with-project-boards/managing-project-boards/linking-a-repository-to-a-project-board
E.g. I cannot find a Menu Icon, Nor can I find a Link Repository Button in the Settings area. How would that be done in Project Beta?
https://support.github.com/ticket/1620144
[image.png]
[image.png]


Thanks,

Paul

Von: Matt Butler ***@***.***
Gesendet: Freitag, 13. Mai 2022 14:48
An: github/feedback
Cc: Paul Leitner; Mention
Betreff: Re: [github/feedback] Invite external collaborator to projects (beta) (Discussion #5874)


@PaulLeitner<https://github.com/PaulLeitner> - you need the collaborator to have access to at least one repo under the account (so that they can see the issues /prs in the project. Does that help?

**Comment by svenvanderburg** [comment]
I am trying to invite members of https://github.com/carpentries-incubator/deep-learning-intro to a private project created in the https://github.com/carpentries-incubator organisation, but that doesn't work... 

As @azenMatt mentioned:
_you need the collaborator to have access to at least one repo under the account (so that they can see the issues /prs in the project_

These collaborators have access to https://github.com/carpentries-incubator/deep-learning-intro but I still cannot add them... I am an administrator of the project and can add some users, but not the users that are part of that specific repo.
